feat: support for new mongodbatlas_atlas_user and mongodbatlas_atlas_users data sources (#1432)
mongodb/terraform-provider-mongodbatlas: examples/atlas-user/users-datasource/README.md

# Example - MongoDB Atlas Users Data Source

> This project aims to provide a very straight-forward example for using the Atlas Users Data Source.

Reviewer: ```suggestion
This project provides a very straight-forward example for using the Atlas Users Data Source.
```

> This project aims to provide a very straight-forward example for using the Atlas Users Data Source.

Reviewer 2: ```suggestion
This project aims to provides a straight-forward example for using the Atlas Users Data Source.
```

Variables Required to be set:

> - `public_key`: Atlas public key

Reviewer 2: ```suggestion
- `public_key`: Atlas Programmatic API  public key
```

> - `private_key`: Atlas  private key

Reviewer: same change as above
```suggestion
- `private_key`: Atlas private key
```

> - `private_key`: Atlas  private key

Reviewer 2: ```suggestion
- `private_key`: Atlas Programmatic API private key
```

- `org_id`: Org ID that identifies the organization whose users you want to return
- `project_id`: Project ID that identifies the project whose users you want to return
- `team_id`: Team ID that identifies the team whose users you want to return

> The example demonstrates the three ways this data source can be used to obtain users from an organization, project, or team.

Reviewer: remove passive voice
```suggestion
The example demonstrates the three ways you can use this data source to obtain users from an organization, project, or team.
```

> The example demonstrates the three ways this data source can be used to obtain users from an organization, project, or team.

Reviewer 2: ```suggestion
The example demonstrates the three ways you can use this data source to obtain users from an Atlas organization, project, or team.
```

> For additional documentation, you can reference the following documentation for each use case:

Reviewer: for brevity
```suggestion
For additional documentation, see:
```
